cvat-core/src/session.js

```diff
@@ -674,6 +674,11 @@
                 task: undefined,
             };
 
+            let dirtyFields = {
```

Comment:
Let's just call it ``updated`` (according to ObjectState which is actually also uses dictionary of updated fields)